engine/src/cubos/engine/tools/scene_editor/plugin.cpp

```diff
@@ -0,0 +1,268 @@
+#include <imgui.h>
+#include <misc/cpp/imgui_stdlib.h>
+
+#include <cubos/core/ecs/commands.hpp>
+
+#include <cubos/engine/assets/plugin.hpp>
+#include <cubos/engine/imgui/plugin.hpp>
+#include <cubos/engine/scene/plugin.hpp>
+#include <cubos/engine/tools/asset_explorer/plugin.hpp>
+#include <cubos/engine/tools/entity_selector/plugin.hpp>
+#include <cubos/engine/tools/scene_editor/plugin.hpp>
+
+using cubos::core::ecs::Commands;
+using cubos::core::ecs::Entity;
+using cubos::core::ecs::Read;
+using cubos::core::ecs::Write;
+using cubos::engine::Asset;
+using cubos::engine::Assets;
+using cubos::engine::Scene;
+using cubos::engine::tools::AssetSelectedEvent;
+
+struct SceneInfo;
+
+/// Resource used to store Scene(s) information
```

Comment:
```suggestion
/// @brief Resource used to store Scene(s) information
```

This isn't really picked up by doxygen since its in a source file, but for consistency lets add this.